src/entities/trade.ts

```diff
@@ -68,27 +64,83 @@ export interface BestTradeOptions {
  */
 export class Trade<TInput extends Currency, TOutput extends Currency, TTradeType extends TradeType> {
   /**
-   * The route of the trade, i.e. which pools the trade goes through.
+   * @deprecated Deprecated in favor of 'routes' property. If the trade consists of multiple routes
+   * this will return an error.
+   *
+   * When the trade consists of just a single route, this returns the route of the trade,
+   * i.e. which pools the trade goes through.
    */
-  public readonly route: Route<TInput, TOutput>
+  public get route(): Route<TInput, TOutput> {
+    invariant(this.routes.length == 1, 'MULTIPLE_ROUTES')
+    return this.routes[0].route
+  }
+
+  /**
+   * The routes of the trade, i.e. which routes the trade consists of.
+   */
+  public readonly routes: {
```

Comment:
this feels like we're expanding the word route to mean what used to be trade, can we name it something different

Reply:
routeAmounts?